src/PushThread.cpp

```diff
@@ -1114,14 +1110,13 @@ void *PushThread::run_undetached(void *ptr)
 					try
 					{
 						//	Send it to DB
-						int ret = shared->mdb->event_Attr(cmd->attr_name, cmd->op_code);
-						if(ret < 0)
-						{
-							//TODO
-						}
+						shared->mdb->event_Attr(cmd->attr_name, cmd->op_code);
 					}
 					catch(Tango::DevFailed  &e)
 					{
+						ERROR_STREAM << "PushThread::run_undetached: An was error detected when removing attribute: "
```

Comment:
This error could occur when adding/stopping/pausing or removing an attribute. So this error message can be misleading.
The error occurred because it was impossible to record the fact that this attribute was started/stopped/paused or removed in HDB++ (in the table dedicated to attributes history).